src/main/java/com/sygic/travel/sdk/tours/facade/ToursFacade.kt

```diff
@@ -9,11 +10,20 @@ class ToursFacade internal constructor(
 	private val toursService: ToursService
 ) {
 	/**
-	 * Creates and sends a request to get the Tours.
-	 * @param toursQuery ToursQuery encapsulating data for API request.
+	 * Creates and sends a request to get the Viator Tours.
+	 * @param toursViatorQuery ToursViatorQuery encapsulating data for API request.
 	 */
-	fun getTours(toursQuery: ToursQuery): List<Tour>? {
+	fun getToursViator(toursViatorQuery: ToursViatorQuery): List<Tour>? {
 		checkNotRunningOnMainThread()
-		return toursService.getTours(toursQuery)
+		return toursService.getToursViator(toursViatorQuery)
+	}
+
+	/**
+	 * Creates and sends a request to get the Get Your Guide Tours.
+	 * @param toursGetYourGuideQuery ToursGetYourGuideQuery encapsulating data for API request.
```

Comment:
I would omit the param annotation.

Reply:
(the same in the second method)

Reply:
No added value, clutter. The majority of the line is autogenerated...

Reply:
I'm afraid that I, again, don't understand, so just asking. What is clutter, what is autogenerated? We had been there before I added the second method. Isn't it better to document the params?

Reply:
- `@param toursGetYourGuideQuery ToursGetYourGuideQuery` is autogenerated and has no added value, since this is already information which is present in the source code
- `encapsulating data for API request.` is actually an infromation which is not autogenerated but does not bring any actual value for developer. 
- clutter - the visual clutter make the file harder to read and process
- maintainability - any rename of parameter or the class will in the better case cause the diff in these line, in the worse case it won't be properly changed here.

Reply:
I know what clutter means, I didn't know which part specifically cause clutter in your opinion. 4th point - I don't agree. When you are renaming, you are supposed to use `refactor -> rename` command in Android Studio. It renames the doc arg as well. However I agree with the no information point, I was just confused, cause I would put every param into a documentation - the fact that you get it without a doc doesn't implicate that everybody does.